.github/workflows/ci.yml

```diff
@@ -12,11 +12,11 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
-            java: 11
+            java: 17
             distribution: zulu
```

Comment:
How about using `temurin` or `liberica`? 😄

Reply:
yeah, not sure why zulu was used by Eugene. I remember there was a problem with temurin 8 not being available for the newer ubuntu image or something, or was it for the arm image. Whatever, we can use temurin.